lib/features/give/pages/organization_list_page.dart

```diff
@@ -270,18 +285,101 @@ class OrganizationListPage extends StatelessWidget {
     }
   }
 
-  IconData getIconByType(CollectGroupType type) {
-    switch (type) {
-      case CollectGroupType.church:
-        return FontAwesomeIcons.placeOfWorship;
-      case CollectGroupType.charities:
-        return FontAwesomeIcons.heart;
-      case CollectGroupType.campaign:
-        return FontAwesomeIcons.handHoldingHeart;
-      case CollectGroupType.artists:
-        return FontAwesomeIcons.guitar;
-      default:
-        return FontAwesomeIcons.church;
+  void _buildActionSheet(BuildContext context, String nameSpace) {
```

Comment:
also good candidate for separate widget I think, especially with platform specific code inside